src/modules/Drag&Drop.js

```diff
@@ -0,0 +1,72 @@
+import { LocalStorage } from "./local-storage";
+import { DisplayList } from "../index"
+
+
+// ADD EVENT TO DRAG ELEMENTS
+  // pseudo code ...
+  // I will take the element and I change the id of the element for the id of the container
+  // I add one to all the elements below the new element
+  
+  function changeTask() {
```

Comment:
kindly. In this line, I recommend using the function Arrow. This helps you to have your code more tidy and short.
and too  My recommendation is not to use a console.log. Remember to use best practices for coding.